content/en/docs/tasks/administer-cluster/migrating-from-dockershim/updrading-and-configuring-cni-plugins.md

```diff
@@ -0,0 +1,155 @@
+CNI version. Otherwise, your network will break.
+
+This issue applies to the following container runtimes:
+- **containerd** v1.6.0 and later
+- **CRI-O** 1.24 and later
+
+## {{% heading "prerequisites" %}}
+
+You must have an existing cluster. This page is about upgrading from Kubernetes
+{{< skew prevMinorVersion >}} to Kubernetes {{< skew latestVersion >}}. If your
+cluster is not currently running Kubernetes {{< skew prevMinorVersion >}} then
+please check the documentation for the version of Kubernetes that you plan to
+upgrade to.
+
+## High-level steps
+
+At a high level, to upgrade and configure the CNI plugins, you typically perform
+the following steps:
+
+1. [Safely drain and cordon the node](/docs/tasks/administer-cluster/safely-drain-node/).
+2. After stopping your container runtime and kubelet services, perform the
+following upgrade operations:
+  - If you’re running CNI plugins, upgrade them to the latest version.
+  - If you’re using non-CNI plugins, replace them with CNI plugins. Use the
+  latest version of the plugins.
+  - Update the plugin configuration file to specify/match the version of the CNI
+  specification that the plugin supports.
+  - For `containerd`, ensure that you have installed the latest version (v1.0.0
+  or later) of the CNI loopback plugin.
+  - Upgrade node components (for example, the kubelet) to Kubernetes 1.24
+  - Upgrade to or install the most current version of the container runtime.
+3. Bring the node back into your cluster by restarting your container runtime
+and kubelet. Uncordon the node.
+
+How you perform these steps varies depending on your cluster setup and
+management tools.
+
+If you performed the upgrade without following these steps and your network is
+broken, see the following [Troubleshooting](#troubleshooting) section.
+
+## Explanation
+
+CNI enables Kubernetes 1.24 networking. The given container runtimes support the
+most current versions of CNI, v1.0.0 and later. For CNI to work, you must use a
+CNI plugin, and its configuration file must specify the CNI version for which
+the plugin was compiled.
+
+If the version of the plugin is not specified, CNI v1.0.0 determines that the
+plugin is v0.1.0, even if this is not the version. As a DevOps engineer or
+administrator, you must review your CNI configuration files to ensure that they
+specify the correct plugin version. If an incorrect version is specified, errors
+might occur.
+
+You must safely drain and cordon a node before upgrading the CNI plugins on the
+node to a current version. Draining is required because the currently running
+pods have a networking setup based on the older CNI plugins. To correctly tear
+down the network on the running pods, you should leave the older version of the
+CNI Plugins in place to perform the cordoning operation correctly.
+
+## An example containerd configuration file
+
+The following example shows a configuration for `containerd` runtime v1.6.x,
```

Comment:
I think that 1.0.0 is backwards compatible and 0.4.0 plugins will work as well

Reply:
@aojea
Please use the PR for the Blog https://github.com/kubernetes/website/pull/33240 for all discussions, not this PR. That way, you're seeing the latest content. I'll migrate all changes from that PR to this one (which has nearly the same content) once it has been reviewed and approved.

Reply:
Yep, sorry